DEVELOPER.md

```diff
@@ -54,6 +54,13 @@ M3 has an extensive, and ever increasing, set of tests to ensure we are able to
 
 Once you have identified a change you want to make, and gathered consensus by talking to some devs, go ahead and make a branch with the changes. To test your changes:
 
+(0) If necessary, generate GoMock files
```

Comment:
Maybe directly below this heading: "If you have updated an interface that has been mocked, you need to update the generated gomock files." to make it more clear?

TY for adding this.

Reply:
Fixed